run_tests.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 
-. ./tests/utils --source-only
-. ./src/kwio.sh --source-only
+. ./src/kw_include.sh --source-only
+include "./tests/utils"
+include "./src/kwio.sh"
```

Comment:
Please, change this in your next patch.

Reply:
@alan-barzilay why did you ignore all the single-quote comments?

Reply:
I wanted to talk about this on our meeting tomorrow, I was planning on arguing that we should keep it as is. Also I'm lazy and was leaving it for last because its going to be a boring manual process haha